### PR TITLE
Remove deprecations in preparation for 1.0

### PR DESCRIPTION
**Pull request by segiddins**
Closes https://github.com/CocoaPods/CocoaPods/issues/3632

### CODE REVIEW

File: lib/cocoapods/installer/user_project_integrator/target_integrator.rb [hunk 1]
```diff
@@ -75,31 +74,6 @@ def inspect
         # @!group Integration steps
         #---------------------------------------------------------------------#
 
-        # Fixes the paths of the copy resource scripts.
-        #
-        # @return [Bool] whether any changes to the project were made.
-        #
-        # @todo   This can be removed for CocoaPods 1.0
-        #
-        def update_to_cocoapods_0_34
```
Comment (kylef) [inline lib/cocoapods/installer/user_project_integrator/target_integrator.rb]: I think we should leave this in, at least until the minimum version of the master specs repository is above 0.34.

Comment (segiddins) [inline lib/cocoapods/installer/user_project_integrator/target_integrator.rb]: hm. when will that be?

Comment (kylef) [inline lib/cocoapods/installer/user_project_integrator/target_integrator.rb]: I wanted to do it a while back but you didn't agree...

Comment (segiddins) [inline lib/cocoapods/installer/user_project_integrator/target_integrator.rb]: ok, I'm bumping it right now then :)

Comment (segiddins) [inline lib/cocoapods/installer/user_project_integrator/target_integrator.rb]: Bumped!

**Pull request closed without merging**